pkg/torcx/paths.go

```diff
@@ -26,11 +26,14 @@ const (
 	DefaultBaseDir = "/var/lib/torcx/"
 	// DefaultConfDir is the default torcx config directory
 	DefaultConfDir = "/etc/torcx/"
+
 	// VendorStorePath is the vendor store path
-	VendorStorePath = VENDOR_DIR + "/store/"
+	VendorStorePath = VendorDir + "store/"
+	// VendorProfile is the vendor (lower) profile
+	VendorProfile = VendorDir + VendorProfileName + ".json"
```

Comment:
I put `vendor.json` in `/usr/share/torcx/profiles`, which is the path returned by `ProfileDirs`, but this looks at `/usr/share/torcx/vendor.json`.  Inserting `"profiles/"` into this line makes it work.  Which path should it be?